tests/unit/algo/plugins/test_plugins.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Test for otx.algo.plugins.xpu_precision"""
+
+
+import pytest
+import torch
+from otx.algo.plugins.xpu_precision import MixedPrecisionXPUPlugin
+from torch.optim import Optimizer
+
+
+class TestMixedPrecisionXPUPlugin:
```

Comment:
Are the new tests you added in this PR needed to run on the XPU devices? If so, please add pytest marker, e.g., xpu, to 
https://github.com/openvinotoolkit/training_extensions/blob/176cdf501f409d537bb12d1055a56bb0a3376eb6/pyproject.toml#L333-L338

Then, please add pytest marker
```suggestion
@pytest.mark.xpu
class TestMixedPrecisionXPUPlugin:
```

We don't have CI pipeline for XPU but adding pytest marker is done before that.

Reply:
it is device agnostic. I patch torch.xpu, so it can be also executed on CUDA.

Reply:
I will mark it as resolved. Please, add more comments if you want me to add smth in the different PR